best-practices/rubyonrails/models/sample.rb

```diff
@@ -30,6 +32,8 @@ def cancel!
   context 'model validations' do
     it { is_expected.to validate_presence_of(:supplier) }
     it { is_expected.to validate_presence_of(:quantity) }
+    it { is_expected.to validate_presence_of(:status) }
+    it { is_expected.to validate_inclusion_of(:status).in_range(ProductStatus.list) }
```

Comment:
Here @felipefontoura are used the `in_array` matcher because in some cases, we can have a enum with these elements: `a: 1, b: 90, c: 1976`, in this case any value between 1 and 1976 are allowed. With `in_array` just the values inside the enum are used I thought.